webapp-django/crashstats/crashstats/static/crashstats/js/socorro/google_analytics.js

```diff
@@ -1,3 +1,3 @@
 window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
-ga('create', 'UA-XXXXX-X', 'auto');
+ga('create', document.documentElement.dataset.googleAnalyticsId, 'auto');
```

Comment:
Out of curiosity, what happens if this node doesn't exist? Does the js error out, but otherwise the page works?

Reply:
Yeah, I think so. I also don't think it's possible to have no `documentElement` in an HTML page (otherwise it'd mean JS was being executed before the HTML was being parsed).